races_bp.py

```diff
@@ -17,13 +17,13 @@ def show_zavody():
 
     nabidka_spolujizdy = []
     if datum_zavodu:
-        zavody = fce_zavody.zavody_dle_datumu(datum_zavodu)
```

Comment:
Looks like you forgot to rename functions and variables in this whole file, except for what got automatically refactored by Pycharm :-)